items.go

```diff
@@ -99,7 +101,21 @@ func (p *Pokemon) UseItem(i *Item) []Transaction {
 func (p *Pokemon) UseMedicine(i *Item) (t []Transaction) {
 	switch i.ID {
 	case ITEM_POTION:
-		t = append(t, p.RestoreHP(20))
+		t = append(t, HealTransaction{Target: p, Amount: 20})
 	}
 	return t
 }
+
+// Uses a vitamin which affects EVs and friendship
+func (p *Pokemon) UseVitamins(i *Item) (t []Transaction) {
+	// https://bulbapedia.bulbagarden.net/wiki/Friendship#In_Generation_IV
+	switch i.ID {
+	case ITEM_CALCIUM:
```

Comment:
do we want to implement vitamins in battle? in the original games, you cannot use vitamins in battle. (see https://bulbapedia.bulbagarden.net/wiki/Protein for an example) the only case in which a Pokemon can usually gain EVs in battle is by knocking out other Pokemon, though EV yield has not yet been generated from the data.

Reply:
is there a list of battle-usable items? i thought I filtered the generated data to only allow those, but maybe it needs more filtering.

Reply:
yeah, there should be one in the miscellaneous section of the wiki